bundles/AdminBundle/Controller/Reports/CustomReportController.php

```diff
@@ -384,7 +384,13 @@ public function createCsvAction(Request $request)
 
         $sort = $request->get('sort');
         $dir = $request->get('dir');
-        $filters = ($request->get('filter') ? json_decode($request->get('filter'), true) : null);
+
+        if ($request->get('filter')) {
+            $filters = json_decode(urldecode($request->get('filter')), true);
+        } else {
+            $filters = null;
+        }
```

Comment:
I think it should stay ternary.

```suggestion
        $filters = $request->get('filter') ? json_decode(urldecode($request->get('filter')), true) : null;
```